app/design/frontend/rwd/default/template/page/html/header.phtml

```diff
@@ -22,7 +22,7 @@
 
         <?= $this->getChildHtml('currency_switcher') ?>
 
-        <p class="welcome-msg"><?= $this->getChildHtml('welcome') ?> <?= $this->getChildHtml('accountLinks') ?></p>
+        <p class="welcome-msg"><?= $this->getChildHtml('welcome') ?>
```

Comment:
is it right to remove accountLinks?

Reply:
That block was only defined in `persistent.xml`, which is no more.

https://github.com/MahoCommerce/maho/blob/43669e44deee5fdba09d4854b5026e756e46abb1/app/design/frontend/base/default/layout/persistent.xml#L16-L20

We could re-define it somewhere like in `page.xml`, but maybe better to just replace it with `$this->getAdditionalHtml()` like is done with the base templates:

https://github.com/MahoCommerce/maho/blob/43669e44deee5fdba09d4854b5026e756e46abb1/app/design/frontend/base/default/template/page/html/header.phtml#L25

Reply:
Disregard last comment, I think actually better to redefine that block in page.xml, will do so now. Thanks for the catch.

Reply:
I think we should go with the implementation from "base" instead of the "accountLinks" which, if I'm not mistaken, seems to exist only in the persistent module with an "emulateAccountLinks" method

Reply:
Yup, after having waffled back and forth, I agree. Will push change after lunch (30 mins) unless you do sooner.